src/TimeModeling/ModelStructure.jl

```diff
@@ -51,6 +51,8 @@ mutable struct Model
     nb::Integer # number of absorbing boundaries points on each side
     m   # slowness squared
     rho # density in g / m^3 (1 for water)
+    Q   # quality factor 
+    f0  # reference frequency for quality factor 
```

Comment:
I still don't like this `f0` floating around everywhere, I'd rather just have `Q/f0` directly, that's cluter that makes things a pain to maintain.

Reply:
Okay I will make this change